docs/v1.1.0/features/logging.md

```diff
@@ -80,7 +71,6 @@ See [configuration](#configuration) for more details.
 To use this middleware, add the following line of code in the `Startup.Configure` method:
 
 ```csharp
-<<<<<<< HEAD
```

Comment:
Merge conflict

Reply:
I know, it was probably miss merged before, but I fixed it here.
You can see it here on master: https://github.com/arcus-azure/arcus.webapi/blob/master/docs/v1.1.0/features/logging.md